modules/learn/pages/clusters-and-availability/automatic-failover.adoc

```diff
@@ -19,26 +20,16 @@ For information on managing auto-failover, see the information provided for Couc
 
 == Failover Events
 
-Auto-failover occurs in response to failover events.
-Events are of three kinds:
-
-* _Node failure_.
-A server-node within the cluster is unresponsive (due to a network failure, out-of-memory problem, or other node-specific issue).
-* _Disk read/write failure_.
-Attempts to read from or write to disk on a particular node have resulted in a significant rate of failure, for longer than a specified time-period.
-The node is removed by auto-failover, even though the node continues to be contactable.
-* _Group failure_.
-An administrator-defined group of server-nodes within the cluster is unresponsive (perhaps due to a network or power failure that has affected an individual, physical rack of machines, or a specific subnet).
+Auto-failover occurs in response to failover events. A server-node within the cluster is unresponsive (due to a network failure, out-of-memory problem, or other node-specific issue).
+The autofailover can fail over multiple nodes at the same time.
 
 [#auto-failover-constraints]
 == Auto-Failover Constraints
 
 Auto-failover is triggered:
 
-* Only on the occurrence of one event at a time.
-If multiple events are concurrent, auto-failover is not triggered.
+* If the majority of nodes (stem:[(nodes + 1)/2]) in the cluster can form a quorum to initiate failover following the failure of one the nodes.
 * Sequentially, only up to an administrator-specified maximum number of events.
```

Comment:
Change to 'Sequentially, only up to an administrator-specified maximum number of nodes.'